compiler/x/codegen/X86Ops.ins

```diff
@@ -56,6 +56,14 @@ INSTRUCTION(vgnop, vgnop,
             BINARY(VEX_L___, VEX_vNONE, PREFIX___, REX__, ESCAPE_____, 0x00, 0, ModRM_NONE, Immediate_0),
             PROPERTY0(0),
             PROPERTY1(IA32OpProp1_PseudoOp)),
+INSTRUCTION(ud2, ud2,
+            BINARY(VEX_L___, VEX_vNONE, PREFIX___, REX__, ESCAPE_0F__, 0x0b, 0, ModRM_NONE, Immediate_0),
+            PROPERTY0(0),
+            PROPERTY1(0)),
+INSTRUCTION(int3, int3,
```

Comment:
The opcode should be upper case but the mnemonic can be lower case like the other instructions.